### DISCUSSION TITLE
Hope to add hover long press trigger preview

### DISCUSSION THREAD

**Discussion by ghost** [u-Sir/peek-pop#19] [Q&A]
Suggest adding mouse long press support for hover preview to avoid accidental triggering of preview

**Comment by u-Sir** [comment]
will added in next version

**Comment by u-Sir** [comment]
[peek_pop-0.0.27-chromium-demo6.zip](https://github.com/user-attachments/files/17088414/peek_pop-0.0.27-chromium-demo6.zip)
[peek_pop-0.0.27-firefox-demo6.zip](https://github.com/user-attachments/files/17088416/peek_pop-0.0.27-firefox-demo6.zip)


you can use this demo version now.  welcome to feedback

**Comment by u-Sir** [comment]
Now available in version 0.0.36. You can update the add-on from Chrome and Firefox.
